src/core/dbus-execute.c

```diff
@@ -120,6 +120,35 @@ static int property_get_oom_score_adjust(
         return sd_bus_message_append(reply, "i", n);
 }
 
+static int property_get_coredump_filter(
+                sd_bus *bus,
+                const char *path,
+                const char *interface,
+                const char *property,
+                sd_bus_message *reply,
+                void *userdata,
+                sd_bus_error *error) {
+
+        ExecContext *c = userdata;
+        uint64_t n;
+
+        assert(bus);
+        assert(reply);
+        assert(c);
+
+        if (c->coredump_filter_set)
+                n = c->coredump_filter;
+        else {
+                _cleanup_free_ char *t = NULL;
+
+                n = 0;
```

Comment:
using COREDUMP_FILTER_DEFAULT as default makes more sense, no?

Reply:
I copied this from property_get_oom_score_adjust() above, but yeah, COREDUMP_FILTER_DEFAULT is probably the least bad choice.